src/HexMaster.Functions.JwtBinding/TokenValidator/TokenValidatorService.cs

```diff
@@ -47,14 +47,15 @@ public class TokenValidatorService
             try
             {
                 var handler = new JwtSecurityTokenHandler();
+                handler.InboundClaimTypeMap.Clear();
```

Comment:
This line was added because dotnet maps claims to Ws-* claims by default. The inbound claim type map needs to be cleared to get the expected claims (as they are in the JWT) instead of these legacy claims.